### ISSUE TITLE
Negative pillarbox

### ISSUE THREAD

**Issue by JamesDAdams** [Aidoku/Aidoku#220]
Hi,

its possible to set pillarbox in negative for zoom on page ?

Example, for reduce white bar : 
![Uploading 883F413B-2524-4945-8A1A-04C5A5DC95BD.png…]()

**Comment by Skittyblock** [comment]
yeah, tachi has this feature and it should be doable to implement auto-detecting borders and zooming pages in.

**Comment by Skittyblock** [comment]
crop borders feature was implemented
